src/main.js

```diff
@@ -3,6 +3,7 @@ export { default as ActionBar } from './components/ActionBar';
 export { default as ActionItem } from './components/ActionItem';
 export { default as NavigationButton } from './components/NavigationButton';
 export { default as Label } from './components/Label';
+export { default as Frame } from './components/Frame';
```

Comment:
As we discussed avoid exports from main.js

Reply:
export { default as Frame } from './components/Frame'; has been removed